actors/miner/src/state.rs

```diff
@@ -1264,9 +1265,9 @@ pub struct MinerInfo {
 
 impl MinerInfo {
     pub fn new(
-        owner: Address,
-        worker: Address,
-        control_addresses: Vec<Address>,
+        owner: ActorID,
```

Comment:
Where are we tracking the migration?

Reply:
@ZenGround0 There shouldn't be a migration needed here -- this is an internal code refactor, not an actual state change. The inputs supplied by users, as well as the  state written to the store remain the same.

Reply:
nice my bad